lore/io/connection.py

```diff
@@ -188,15 +190,21 @@ def insert(self, table, dataframe, batch_size=10 ** 5):
                     tmp = tempfile.NamedTemporaryFile(delete=False, suffix='.csv.gz')
                     tmp.close()
                     batch.to_csv(tmp.name, index=False, header=False, sep='|', na_rep='\\N', quoting=csv.QUOTE_NONE, compression='gzip')
-                    self._connection.connection.cursor().execute('PUT file://%(path)s @~/staged;' % {'path': tmp.name})
+                    letters = string.ascii_letters
+                    suffix = ''.join(random.choice(letters) for i in range(20))
```

Comment:
Why not use a real uuid like `uuid.uuid4()`?

Reply:
ohh didn't know there was something like that available. let me modify it.